.github/workflows/reusable-ci-workflows.yml

```diff
@@ -0,0 +1,77 @@
+name: Python CI Reusable Workflow
+
+on:
+  workflow_call:
+    inputs:
+      python-version:
+        required: true
+        type: string
+      ref:
+        required: false
+        type: string
+
+env:
+  PYLINT_MIN_SCORE: "9.00"
+  COVERAGE_MIN_PERC: "90"
+  COMPLEXITY_MAX_SCORE: "30"
+  SOURCE_DIRS: "qmi/"
+  BADGES_DIR: ".github/badges"
```

Comment:
can we move the `BADGES_DIR` env to the `pull-request-ci` since it is not reusable?

Reply:
Moved.